src/lib/core/components/visualizations/InputVariables.tsx

```diff
@@ -180,22 +181,27 @@ export function InputVariables(props: Props) {
   return (
     <div>
       <div className={classes.inputs}>
-        {inputs.map((input, index) => (
-          <div key={input.name} className={classes.input}>
-            <div className={classes.label}>{input.label}</div>
-            <VariableTreeDropdown
-              rootEntity={entities[0]}
-              disabledVariables={disabledVariablesByInputIndex[index]}
-              starredVariables={starredVariables}
-              toggleStarredVariable={toggleStarredVariable}
-              entityId={values[input.name]?.entityId}
-              variableId={values[input.name]?.variableId}
-              onChange={(variable) => {
-                handleChange(input.name, variable);
-              }}
-            />
-          </div>
-        ))}
+        {inputs.map((input, index) => {
+          const Wrapper = input.wrapper ?? (({ children }) => <>{children}</>);
```

Comment:
Is there a more elegant way to do the 'no op' wrapper?

Reply:
If you adopt my suggestion of giving `InputSpec` an `InputComponent`, this line could be changed to something like this:

```suggestion
          const InputComponent = input.InputComponent ?? VariableTreeDropdown;
```